lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -41,16 +43,31 @@ class UserProfileBloc extends Cubit<UserProfileState> with HydratedMixin {
       );
     }
     emit(profile);
-    registerForNotifications();
+    final injector = ServiceInjector();
+    final breezLib = injector.breezSDK;
+    breezLib.nodeStateStream.firstWhere((nodeState) => nodeState != null).then((_) {
+      registerForNotifications(breezLib);
+    });
   }
 
-  Future registerForNotifications() async {
+  Future registerForNotifications(BreezSDK breezLib) async {
     _log.info("registerForNotifications");
     String? token = await _notifications.getToken();
     if (token != null) {
       if (token != state.profileSettings.token) {
```

Comment:
We should register anyway regardless if this is already registered since it refreshes the last_update timestamp on the server. Server may remove outdated tokens based on timestamp.